src/components/TablePackages/TablePackages.tsx

```diff
@@ -0,0 +1,119 @@
+import { CheckBoxOutlineBlankOutlined } from "@mui/icons-material";
+import {
+  Table,
+  TableContainer,
+  Paper,
+  TableHead,
+  TableRow,
+  TableCell,
+  Typography,
+} from "@mui/material";
+import { Box } from "@mui/system";
+import { useEffect, useState } from "react";
+
+const headers = [
+  "ID listu",
+  "Imię i nazwisko",
+  "Status pakowania",
+  "Kraj",
+  "Miasto",
+  "Data Akceptacji",
+  "",
+];
+
+const mockTableElements: Packages[] = [
+  {
+    id: 1,
+    wishListId: 1,
+    name: "Joe Kernel",
+    kindness: 5,
+    status: "sent",
+    country: "Polska",
+    city: "Kraków",
+    createdAt: "2022-12-15T10:08:52.796000+01:00",
+  },
+  {
+    id: 2,
+    wishListId: 1,
+    name: "YES",
+    kindness: 5,
+    status: "sent",
+    country: "Polska",
+    city: "Kraków",
+    createdAt: "2022-12-15T10:08:52.796000+01:00",
+  },
+];
+
+interface HeadersValues {
+  headers: string[];
+}
+
+interface Packages {
+  id: number;
+  wishListId: number;
+  name: string;
+  kindness: number;
+  status: string;
+  country: string;
+  city: string;
+  createdAt: string;
+}
```

Comment:
Generally it is good idea to keep types in a separate file next to the component. Could you please move those declarations to `types.ts` file?